tests/static/intentManager/printing-intent.xml

```diff
@@ -24,7 +24,7 @@
 		</model>
 		<odkx:intent xmlns:odkx="http://opendatakit.org/xforms" id="print_data" class="org.commcare.dalvik.action.PRINT">
 			<extra key="text" ref="/data/text" />
-			<extra key="cc:print_template_reference" ref="jr://file/commcare/doc/data/print_data.doc" />
+			<extra key="cc:print_template_reference" ref=".">jr://file/commcare/doc/data/print_data.doc</extra>
```

Comment:
Clayton, was planning on talking to you about this later. @phillipm worked on this with us on Thursday and we came up with a workaround that may or may not be acceptable, but it's a much bigger lift on HQ to do it the way you were saying, so we're trying to find a way around it. The 3 of us should chat when you can